source/declarations.tex

```diff
@@ -7115,7 +7115,7 @@
 that refers to the object bound to $\tcode{r}_i$;
 the referenced type is $\tcode{T}_i$.
 The initialization of \tcode{e} and
```

Comment:
missed this one: \tcode{e} -> \exposid{e}

Reply:
fixed